react-infinite/build.boot

```diff
@@ -21,17 +21,17 @@
 
 (deftask package []
   (comp
-   (download :url (str "https://github.com/seatgeek/react-infinite/archive/v" +lib-version+ ".zip")
-             :checksum "3c9e2b831e996b59a19b560be5e9f77a"
+   (download :url (str "https://github.com/seatgeek/react-infinite/archive/" +lib-version+ ".zip")
              :unzip true)
-
-   (sift :move {#"^react-infinite-(.*)/dist/react-infinite.js$" "cljsjs/react-infinite/development/react-infinite.inc.js"
-                #"^react-infinite-(.*)/dist/react-infinite.min.js$" "cljsjs/react-infinite/production/react-infinite.min.inc.js"})
-
+   (sift :move {#"^react-infinite-(.*)/dist/react-infinite.js$"
+                "cljsjs/react-infinite/development/react-infinite.inc.js"
+                #"^react-infinite-(.*)/dist/react-infinite.min.js$"
+                "cljsjs/react-infinite/production/react-infinite.min.inc.js"})
    (sift :include #{#"^cljsjs"})
-
-   (deps-cljs :name "react-infinite"
-              :requires ["react" "react-dom" "prop-types"])
+   (deps-cljs :name "cljsjs.react-infinite"
+              :requires ["cljsjs.react"
+                         "cljsjs.react.dom"
+                         "cljsjs.prop-types"])
    (pom)
-
-   (jar)))
+   (jar)
+   (validate-checksums)))
```

Comment:
Try using `validate` here for foreign lib and JS file validation, in addtion to checksums.